components/os/kmsg_matcher.go

```diff
@@ -11,12 +13,24 @@ const (
 	//
 	// ref.
 	// https://docs.kernel.org/admin-guide/sysctl/fs.html#file-max-file-nr
-	eventVFSFileMaxLimitReached   = "vfs_file_max_limit_reached"
-	regexVFSFileMaxLimitReached   = `VFS: file-max limit \d+ reached`
-	messageVFSFileMaxLimitReached = "VFS file-max limit reached"
+	eventNameVFSFileMaxLimitReached = "vfs_file_max_limit_reached"
+	regexVFSFileMaxLimitReached     = `VFS: file-max limit \d+ reached`
+	messageVFSFileMaxLimitReached   = "VFS file-max limit reached"
+
+	// Kernel panic event constants
+	eventNameKernelPanic = "kernel_panic"
 )
 
-var compiledVFSFileMaxLimitReached = regexp.MustCompile(regexVFSFileMaxLimitReached)
+var (
+	compiledVFSFileMaxLimitReached = regexp.MustCompile(regexVFSFileMaxLimitReached)
+
+	// Kernel panic regex patterns
+	// e.g., Kernel panic - not syncing: hung_task: blocked tasks
+	kernelPanicStartRegexp = regexp.MustCompile(`Kernel panic - not syncing: hung_task: blocked tasks`)
```

Comment:
we can reference this doc, there are some common messages. https://cloud.google.com/compute/docs/troubleshooting/kernel-panic